__tests__/unit/BuildingIdentification.test.js

```diff
@@ -11,5 +11,5 @@ describe("Building Identification component", () => {
     test("number of markers", () => {
         const tree = renderer.create(<BuildingIdentification />).toJSON();
         expect(tree.children.length).toBe(10);
-      });
-});
+    });
+});
```

Comment:
Very minor but please add new line at end of file (coding conventions related).

Reply:
will do!